src/types/Account/methods/broadcastTransaction.js

```diff
@@ -36,9 +36,12 @@ function impactAffectedInputs({ inputs }) {
 /**
  * Broadcast a Transaction to the transport layer
  * @param {Transaction|RawTransaction} transaction - A txobject or it's hexadecimal representation
+ * @param {isBoolean=false} forceBroadcast - Allow to force broadcast on error.
  * @return {Promise<transactionId>}
  */
-async function broadcastTransaction(transaction) {
+async function broadcastTransaction(transaction, forceBroadcast = false) {
```

Comment:
Maybe we do a specific option like `options.skipFeeValiation`?

Reply:
Make sense ! Updated